Classes/Repository/RepositoryClient.swift

```diff
@@ -29,6 +31,12 @@ extension RepoSearchPagesQuery: RepositoryQuery {
             results.search.pageInfo.hasNextPage else { return nil }
         return results.search.pageInfo.endCursor
     }
+    
+    func numberOfItems(from data: GraphQLSelectionSet) -> Int {
+        guard let data = data as? RepoSearchPagesQuery.Data else { return 0 }
```

Comment:
I think it is better to `return nil` rather then `0`. If data is `nil`, doesn't necessary mean that the number of items is 0 and it wouldn't be accurate to show 0.